draft-regext-simple-contact.md

```diff
@@ -88,135 +91,61 @@ The "kind" JSON value is a string, that is either "individual", "role" or
 
     "kind" : "role"
 
+There is no equivalent to the "role" value in either jCard or JSContact,
+though role entities are common in RDAP registries.
 
 ## Names
 
-Names may be expressed as unstructured text and as structured data.
-When names are given as structured data, it is RECOMMENDED to also
-give unstructured names. This may require servers which store names
-in structured data to synthesize the unstructured names. 
-Servers which store only unstructured names SHOULD NOT attempt to synthesize 
-and provide structured names from those values, because of the difficulty 
-of doing this correctly for all types of names.
-
 Names can be expressed for each kind of the entity, as described in the
 "kind" string. When describing an "individual", the name of the individual's
 role and organization may also be expressed. When describing a "role", the
 name of the role's organization may also be expressed. It is NOT RECOMMENDED
 to express the name of a role or individual when the kind is "organization".
```

Comment:
Thinking on this again, for the APNIC server, we would use the single name field that corresponds to the contact's kind value, with the relationship to an organisation being inferred by way of the inclusion of an organisation contact in the 'entities' key on the contact or resource record itself.  Documenting that as an alternative (or additional) approach to connecting an entity to a related entity here might be useful.  What do you think?

Separately, what would be the use case for providing a role/individual name for an organisation contact?

Reply:
I think documenting that as an alternative would be good. We can even ask the WG if it might preferred or the only way. Nesting is more descriptive of the relationships.

I know of no use cases for providing role/individual name for an organization.